### PR TITLE
[event-store] Stream event on/from the filesystem

### LINKED ISSUE #2: Improve file system event store performance by streaming data
We could use this package to help us on the JSONLines stream: https://github.com/LinusU/node-jsonlines

### PR DESCRIPTION
**Pull request by armandabric**
This will mainly improve the fetch history performance.
On write we are not yet fully in a stream mode: this will indroduce concistancy issue to manage. For now we love 💋

Fix #2

### CODE REVIEW

File: packages/event-store/src/EventStore/FileSystemEventStore.js [hunk 1]
```diff
@@ -23,37 +22,55 @@ export default class FileSystemEventStore implements EventStore {
...
+      stringify.on('error', err => reject(err));
+
+      const write = fs.createWriteStream(this.storagePath, {
+        encoding: 'utf8',
+        flags: 'a',
+      });
+      write.on('error', err => reject(err));
+      write.on('finish', () => resolve());
+
+      storableEventsStream.pipe(stringify).pipe(write);
```
Comment (armandabric) [inline packages/event-store/src/EventStore/FileSystemEventStore.js]: 🚬

File: packages/event-store/src/EventStore/FileSystemEventStore.js [hunk 2]
```diff
@@ -23,37 +22,55 @@ export default class FileSystemEventStore implements EventStore {
...
-    return content
-      .split('\n')
-      .map(line => JSON.parse(line))
-      .filter(payload => payload.stream_name === streamName)
-      .map(({ event }) => this.eventSerializer.deserialize(event));
+        history.push(this.eventSerializer.deserialize(storedEvent.event));
+      });
+      parse.on('end', () => resolve(history));
+
+      read.pipe(parse);
```
Comment (armandabric) [inline packages/event-store/src/EventStore/FileSystemEventStore.js]: 🚬

File: packages/event-store/src/EventStore/FileSystemAdapter/JSONLinesFileSystemAdapter.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    (data, encoding, cb) =>
+      data.stream_name === streamName ? cb(null, data) : cb(null, ''),
+  );
+};
+
+const createExtractPayloadStream = () => {
+  return transformify((data, encoding, cb) => cb(null, data.event));
+};
+
+const createEncapsulatePayloadStream = (streamName: string) => {
```
Comment (shouze) [inline packages/event-store/src/EventStore/FileSystemAdapter/JSONLinesFileSystemAdapter.js]: 2 verbs in a single action, is it a good idea?

Comment (armandabric) [reply packages/event-store/src/EventStore/FileSystemAdapter/JSONLinesFileSystemAdapter.js]: This is not perfect. I do this keep concistancy with the node `fs.create[Read|Write]Stream` method. I will simplify this

**Pull request merged**